src/modules/ContextMenu.ts

```diff
@@ -1,34 +1,34 @@
+import CONSTANTS from './constants';
```

Comment:
As a matter of "standard" among the modules in general I have seen that the file in question is always on `scripts / contants.js` or` module / contants.js` . For me it would be better to leave it here.